src/core/src/runtime/itensor.cpp

```diff
@@ -160,7 +160,19 @@ void ITensor::copy_to(const std::shared_ptr<ov::ITensor>& dst) const {
 
     bool finish = false;
     for (size_t dst_idx = 0, src_idx = 0; !finish;) {
-        memcpy(dst_data + dst_idx, src_data + src_idx, src_strides[src_strides.size() - 1]);
+        if (get_element_type() == element::string) {
```

Comment:
Can move this `if` out side loop to not do check every iteration.
Make some helpers which copy strings or other type make if to select function then do the loop.

Reply:
Fixed.